src/components/Board.js

```diff
@@ -12,14 +12,94 @@ class Board extends Component {
     super();
 
     this.state = {
-      cards: [],
+      cards: []
     };
+
+  }
+
+  componentDidMount() {
+    console.log("The component did in fact mount");
+    const GET_ALL_CARDS_URL = "https://inspiration-board.herokuapp.com/boards/lindsay/cards";
```

Comment:
it would be nice if you used the `prop`s passed into `Board` from `App` -- `url` and `boardName`!